core/App/screens/ProofRequest.tsx

```diff
@@ -223,32 +240,41 @@ const ProofRequest: React.FC<ProofRequestProps> = ({ navigation, route }) => {
     })
   }
 
+  const connectionImage = getConnectionImageUrl(proof?.connectionId)
```

Comment:
In the previous file you used `??` to coalesce this to and empty string but not here. I think using one or the other is better.